src/beautifiers/phpcbf.coffee

```diff
@@ -11,12 +11,10 @@ module.exports = class PHPCBF extends Beautifier
   isPreInstalled: false
 
   options: {
-    _:
-      standard: ["standard", (standard) ->
-        if (standard) then \
-          standard else "PEAR"
-      ]
-    PHP: true
+    PHP:
+      phpcbf_path: true
+      phpcbf_version: true
+      standard: true
```

Comment:
If PHPCS is called without a standard the PEAR standard is used any ways. If we want to default the standard to PEAR I feel it would be better to use the default property in options